src/librustc_mir/borrow_check/places_conflict.rs

```diff
@@ -17,17 +17,45 @@ use rustc::mir::{Projection, ProjectionElem};
 use rustc::ty::{self, TyCtxt};
 use std::cmp::max;
 
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
```

Comment:
Documentation request: what does it mean to "bias" a place conflict? Unfortunately I don't find any answer to this question in the variant names. I'll see if I can infer something from reading the code, but a follow-up commit with docs on this enum might be useful.

Reply:
Added a comment for this.